### DISCUSSION TITLE
v1.13.12

### DISCUSSION THREAD

**Discussion by deividAlfa** [deividAlfa/stm32_soldering_iron_controller#133] [Announcements]
- Dedicated Stand input where supported (T12-958 v2 has this).
- Lower default PID values, existing ones were too agressive for certain tips, causing oscillation.
- T12-958 v2: Fix default encoder rotation direction, add stand input pin.
- Set NTC beta to 3950 by default (Instead 3450).

<hr /><em>This discussion was created from the release <a href='https://github.com/deividAlfa/stm32_soldering_iron_controller/releases/tag/'>v1.13.12</a>.</em>

**Comment by WerrW01f** [comment]
Please explain what caused the transition to a different type of thermistor?
> * Set NTC beta to 3950 by default (Instead 3450).
